src/main/scala/nl/knaw/dans/easy/pid/PidGeneratorService.scala

```diff
@@ -33,7 +33,7 @@ class PidGeneratorService(val serverPort: Int, app: PidGeneratorApp) extends Deb
         override def probeForCycleClass(classLoader: ClassLoader): (String, LifeCycle) = {
           ("pid-lifecycle", new LifeCycle {
             override def init(context: ServletContext): Unit = {
-              context.mount(new PidGeneratorServlet(app), "/")
+              context.mount(new PidGeneratorServlet(app), "/pids")
```

Comment:
we probably deleted this `pids` last week, but we need it here to comply with the interface that is used in e.g. easy-ingest-flow